src/python/pants/backend/project_info/regex_lint.py

```diff
@@ -226,33 +221,19 @@ def __init__(self, config: ValidationConfig):
         self._content_matchers = {cp.name: ContentMatcher(cp) for cp in config.content_patterns}
         self._required_matches = config.required_matches
 
-    def check_source_file(self, path, content):
-        content_pattern_names, encoding = self.get_applicable_content_pattern_names(path)
-        matching, nonmatching = self.check_content(content_pattern_names, content, encoding)
-        return RegexMatchResult(path, matching, nonmatching)
-
-    def check_content(self, content_pattern_names, content, encoding):
-        """Check which of the named patterns matches the given content.
```

Comment:
Now that there are proper type hints, the docstring felt redundant - the code is self documenting.

Reply:
Agreed. The comment just recapitulates the name and signature of the method...